tuning-crc-openshift-cluster/update-kube-controlplane.sh

```diff
@@ -0,0 +1,104 @@
+
+SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i $SSH_KEYS_OF_MASTER_NODE"
+SCP="scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i $SSH_KEYS_OF_MASTER_NODE"
+
+JQ=${JQ:-jq}
+YQ=${YQ:-yq}
+
+ update_kube_apiserver_manifests() {
+    ${SSH} core@master -- sudo cat $1  > current_manifest.json
+    new_memory_value=$2
+    new_cpu_value=$3
+    ${JQ}  --arg new_memory_value "$new_memory_value" '(.spec.containers[].resources.requests.memory) |= $new_memory_value' current_manifest.json > updated_wth_memory_manifest.json
+    ${JQ}  --arg new_cpu_value "$new_cpu_value" '(.spec.containers[].resources.requests.cpu) |= $new_cpu_value' updated_wth_memory_manifest.json > updated_wth_cpu_manifest.json
+    ${JQ} '.spec.containers[].env |= . + [{"name": "GOGC", "value": "5"}, {"name": "GODEBUG", "value": "madvdontneed=1"}] ' updated_wth_cpu_manifest.json > updated_with_env_manifest.json
+    new_kubeapi_cpu_value=$4
+    ${JQ}  --arg new_kubeapi_cpu_value "$new_kubeapi_cpu_value" '(.spec.containers[] | select(.name == "kube-apiserver") | .resources.requests.cpu) |= $new_kubeapi_cpu_value' updated_with_env_manifest.json  > final_manifest.json
+    cat final_manifest.json | ${JQ} -c > unformatted_final_manifest.json
+
+    ${SCP} -r unformatted_final_manifest.json core@master:/home/core/updated-kube-apiserver-manifest.yaml
+    ${SSH} core@master -- sudo cp /home/core/updated-kube-apiserver-manifest.yaml $1
+    ## Remove temp. files created
+    rm current_manifest.json updated_wth_memory_manifest.json updated_wth_cpu_manifest.json updated_with_env_manifest.json final_manifest.json unformatted_final_manifest.json
+}
+
+ update_kube_controller_manifests() {
+    ${SSH} core@master -- sudo cat $1  > current_manifest.json
```

Comment:
Not sure why you are using `core@master` instead use the ` core@api.${CRC_VM_NAME}.${BASE_DOMAIN} ` which is used in available scripts.